README.md

```diff
@@ -1,48 +1,15 @@
-
-Additional features will arrive during the beta period until we reach general availability (GA). We will be cutting versioned releases at least every two weeks starting in the first release. There will also be nightly builds based off the latest code in master.
-
-Once we close on the final feature set of what will be in the first GA release of InfluxDB in the 2.x line, we will move into the release candidate (RC) phase. At that point, we do not expect there to be breaking changes to the API or Flux language. We may still need to make a breaking change prior to GA due to some unforseen circumstance, but it would need to be extremely important and will be clearly communicated via the changelog and all available channels.
-
-Our current plans are to release RCs suitable for production usage, but we will re-evaluate in consultation with the community as the cycle progresses. During the RC period, we will focus on feedback from users, bug fixes, performance, and additive features (where time permits).
-
-### What you can expect in the Beta and RC Phases
-
-#### Beta
-**Releases every two weeks or as needed**
-
-Planned additions include:
-- Compatibility layer with 1.x including: 1.x HTTP Write API  and HTTP Read API support for InfluxQL
-- Import Bulk Data from 1.x - convert TSM from 1.x to 2.x
-- Performance tuning, stability improvements, and fine tuning based on community feedback.
-- Finalization of supported client libraries starting with JavaScript and Go.
-
-#### RC
-**As needed**
-
-Planned activities include:
-- Performance tuning, stability improvements, and fine-tuning based on community feedback.
-
-### What is **NOT** planned?
-- Migration of users/security permissions from InfluxDB v1.x to 2.x.  ACTION REQUIRED: Re-establish users and permissions within the new unified security model which now spans the underlying database and user interface.
-- Migration of Continuous Queries.  ACTION REQUIRED: These will need to be re-implemented as Flux tasks.
-- Direct support by InfluxDB for CollectD, StatsD, Graphite, or UDP.  ACTION REQUIRED: Leverage Telegraf 1.9+ along with the InfluxDB v2.0 output plugin to translate these protocols/formats.
-
 ## Installing from Source
 
-We have nightly and weekly versioned Docker images, Debian packages, RPM packages, and tarballs of InfluxDB available at the [InfluxData downloads page](https://portal.influxdata.com/downloads/). We also provide the `influx` command line interface (CLI) client as a separate binary available at the same location.
+We have nightly and versioned Docker images, Debian packages, RPM packages, and tarballs of InfluxDB available at the [InfluxData downloads page](https://portal.influxdata.com/downloads/). We also provide the `influx` command line interface (CLI) client as a separate binary available at the same location.
```

Comment:
nightly versioned?

Reply:
we have both nightly and versioned images. the nightly isn't versioned, other than with a git hash.